lib/private/RedisFactory.php

```diff
@@ -54,13 +62,26 @@ private function create() {
 			} else {
 				$readTimeout = null;
 			}
-			$this->instance = new \RedisCluster(null, $config['seeds'], $timeout, $readTimeout);
+			if (isset($config['connection_parameters'])) {
+				if (!$isConnectionParametersSupported) {
+					throw new \UnexpectedValueException(\sprintf('php-redis extension must be equal %s or higher to support connection parameters',
```

Comment:
```suggestion
					throw new \UnexpectedValueException(\sprintf('php-redis extension must be version %s or higher to support connection parameters',
```

It will read nicely to just say "php-redis extension must be version 5.3.0 or higher..."